app/src/main/java/com/android/unio/model/firestore/transform/Serialization.kt

```diff
@@ -31,7 +31,8 @@ fun UserRepositoryFirestore.Companion.serialize(user: User): Map<String, Any> {
       "joinedAssociations" to user.joinedAssociations.list.value.map { it.uid },
       "interests" to user.interests.map { it.name },
       "socials" to user.socials.map { mapOf("social" to it.social.name, "content" to it.content) },
-      "profilePicture" to user.profilePicture)
+      "profilePicture" to user.profilePicture,
+      "savedEvents" to user.savedEvents.list.value.map { it.uid })
```

Comment:
Just so you know when merging with main you'll be able to change this line to the more succint: `user.savedEvents.uids`.